pymongoexplain/commands.py

```diff
@@ -24,25 +24,32 @@
 Document = Union[dict, SON]
 
 class BaseCommand():
-    def __init__(self, dictionary):
-        if type(self) == BaseCommand:
-            raise NotImplementedError
-        self.command_document = self.convert_to_camelcase(dictionary)
-
-    def convert_to_camelcase(self, d: dict) -> dict:
+    def __init__(self, command_name, collection):
+        self.command_document = {}
+        self.command_name = command_name
+        self.collection = collection
+
+    def convert_to_camelcase(self, d, exclude_keys=[]):
```

Comment:
What is the value of having this be a method on BaseCommand?
I think it makes more sense to create a `utils.py` submodule and add this to that as a standalone function. That way it is usable elsewhere, like in test code

Reply:
Done.